flink-test-utils-parent/flink-connector-testing/src/main/java/org/apache/flink/connectors/test/common/junit/annotations/Context.java

```diff
@@ -40,4 +41,4 @@
 @Target(ElementType.FIELD)
 @Retention(RetentionPolicy.RUNTIME)
 @Experimental
-public @interface ExternalContextFactory {}
+public @interface Context {}
```

Comment:
I suggest add a qualifier for `Context`, `Context` looks like a top level interface for all `xxContext`